public_html/wp-content/themes/wordcamp-central-2012/template-home.php

```diff
@@ -105,7 +107,23 @@
 			</a>
 		</div><!-- .wc-news -->
 
-		<div class="wc-tweets last">
+		<div class="wc-global-sponsors last">
+			<h3><strong>Some Of Our Sponsors</strong></h3>
+
+			<?php
+			echo wp_kses_post( MES_Block\render( array(
+				'number'      => 3,
+				'region_id'   => 'all',
+				'level_id'    => 3040794,
+				'image_align' => 'center',
+				'image_width' => 300,
```

Comment:
🤔 I assumed that the output of `render_featured_image` would include a `srcset` attribute, since it's using `get_the_post_thumbnail` under the hood. But you're right that it's not doing it in this case. Maybe my assumption is wrong? Looking into it.

Reply:
yeah, that's interesting— i wonder if it's a theme-support thing, since this is based on 2010?

Reply:
After some digging, I suspect it's because things go through multiple layers of `wp_kses_post`. Astonishingly, `srcset` and `sizes` are not considered valid attributes on the `img` tag with that set of allowed tags...

Reply:
Looks related: https://core.trac.wordpress.org/ticket/29807

Reply:
I guess for now we can just double the set width 😕

Reply:
8f1e1c7e